spring-cloud-kubernetes-config/src/main/java/org/springframework/cloud/kubernetes/config/ConfigMapPropertySource.java

```diff
@@ -98,7 +98,7 @@ private static String getName(KubernetesClient client, String name,
 		}
 		catch (Exception e) {
 			LOG.warn("Can't read configMap with name: [" + name + "] in namespace:["
-					+ namespace + "]. Ignoring");
```

Comment:
You don't need to add "Reason for failure" just pass the exception

Reply:
Done